File: 2949.py
n = int(input())
hobbit = humano = elfo = anao = mago = 0
for i in range(0, n):
    h, b = input().split()
    if b == 'X':
        hobbit += 1
    if b == 'H':
        humano += 1
    if b == 'E':
        elfo += 1
    if b == 'A':
        anao += 1
    if b == 'M':
        mago += 1
print(f'''{hobbit} Hobbit(s)
{humano} Humano(s)
{elfo} Elfo(s)
{anao} Anao(s)
{mago} Mago(s)''')

File: 2949.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var n = Number(lines.shift())
var hobbit = 0;var humano = 0;var elfo = 0;var anao = 0;var mago = 0;
for(var i = 0;i < n; i++){
    var k = lines.shift().split(" ")
    var b = k[1]
    if(b === 'X'){
        var hobbit = hobbit + 1}
    else if(b === 'H'){
        var humano = humano + 1}
    else if(b === 'E'){
        var elfo = elfo + 1}
    else if(b === 'A'){
        var anao = anao + 1}
    else if(b === 'M'){
        var mago = mago + 1}}
console.log(`${hobbit} Hobbit(s)
${humano} Humano(s)
${elfo} Elfo(s)
${anao} Anao(s)
${mago} Mago(s)`)
